php/libraries/NDB_Page.class.inc

```diff
@@ -280,6 +280,27 @@ class NDB_Page
     }
 
     /**
+     * Wrapper to create a radio button
+     *
+     * @param string $name     The field name of this radio button
+     * @param string $label    The label to attach to this radio button
+     * @param array  $options  Options to pass to QuickForm for this
+     *                         radio button
+     * @param array  $optional Optional extra HTML attributes to add
+     *                         to the radio button
+     *
+     * @return none
+     */
+    function addRadio($name, $label, $options, $optional=array())
```

Comment:
I would have this work in the sense that I pass in all the values of the radio buttons, looping through that and adding each as their own radio button to the group, adding the group at the end. This would remove the need of something like this
```
 $pendGroup[] = $this->createRadio("Pending_approval", "Yes", array('value'=>'Y'));
 $pendGroup[] = $this->createRadio("Pending_approval", "No", array('value'=>'N'));
 $this->addGroup($pendGroup, 'Pending_approval_Group', 'Pending Approval:', $this->_GUIDelimiter, false);
```
the following would be easier to read and would require less repetitive code
```
$this->addRadio($name, $groupLabel,array([0]=>array($label=>$value)...), $optional)
```